build.gradle.kts

```diff
@@ -98,6 +95,11 @@ sourceSets {
 
 springBoot { mainClass.set("it.pagopa.wallet.WalletApplicationKt") }
 
+tasks.create("applySemanticVersionPlugin") {
+  dependsOn("prepareKotlinBuildScriptModel")
+  apply(plugin = "com.dipien.semantic-version")
+}
+
```

Comment:
Why is this needed?

Reply:
Because plugin listed into plugin section are automatically applied, but by then the version is not already set making the plugin fail blaming about version to be set in root project.
I then set to false plugin apply into plugins section and apply it later after all the project variables have been set